From 5150ff6e027ffa724492de372814794fe7d7ebd6 Mon Sep 17 00:00:00 2001
From: Chen, Chien-Chia <machen@novell.com>
Date: Thu, 9 Dec 2010 10:52:08 +0800
Subject: [PATCH 027/244] Staging: rt2860: fix previous patch error

Somehow Greg messed up the last patch and missed a chunk.  This patch
contains the missing chunk.

Acked-by: Lee, Chun-Yi <jlee@novell.com>
Signed-off-by: Chen, Chien-Chia <machen@novell.com>
Cc: stable <stable@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/rt2860/chips/rt30xx.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/drivers/staging/rt2860/chips/rt30xx.c
+++ b/drivers/staging/rt2860/chips/rt30xx.c
@@ -441,7 +441,7 @@ void RT30xxReverseRFSleepModeSetup(struc
 
 		/* VCO_IC, RF R7 register Bit 4 & Bit 5 to 1 */
 		RT30xxReadRFRegister(pAd, RF_R07, &RFValue);
-		RFValue |= 0x30;
+		RFValue |= 0x20;
 		RT30xxWriteRFRegister(pAd, RF_R07, RFValue);
 
 		/* Idoh, RF R9 register Bit 1, Bit 2 & Bit 3 to 1 */
